parsely/src/test/java/com/parsely/parselyandroid/FlushManagerTest.kt

```diff
@@ -1,49 +1,48 @@
 package com.parsely.parselyandroid
 
-import androidx.test.core.app.ApplicationProvider
 import kotlin.time.Duration.Companion.seconds
 import kotlinx.coroutines.ExperimentalCoroutinesApi
 import kotlinx.coroutines.test.advanceTimeBy
 import kotlinx.coroutines.test.runCurrent
 import kotlinx.coroutines.test.runTest
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.Test
-import org.junit.runner.RunWith
-import org.robolectric.RobolectricTestRunner
 
 @OptIn(ExperimentalCoroutinesApi::class)
-@RunWith(RobolectricTestRunner::class)
 class FlushManagerTest {
 
     private lateinit var sut: FlushManager
-    private val tracker = FakeTracker()
+    private var flushEventsCounter = 0
```

Comment:
Do we need these properties, or would it be possible to use local variables instead? I skimmed over the tests and I can't see the need for the properties, but I might be missing something.